draft-ietf-tls-esni.md

```diff
@@ -535,6 +532,26 @@ application using (D)TLS or externally configured on both sides,
 implementations MUST compute the field as specified in
 {{encrypted-client-hello}}.
 
```

Comment:
- Let's promote this subsubsection to a subsection and make it a subsection of {{encrypted-client-hello}}
- In order to be consistent with the other subsections, I suggest using a title that's suggestive of the code points purpose. E.g., `Differentiating between the ClientHelloInner and the ClientHelloOuter`.

Reply:
> make it a subsection of {{encrypted-client-hello}}

I'm not sure about this because `ech_is_inner` isn't part of the `encrypted_client_hello` extension.

Reply:
I'd leave it as-is for now. We can move it around later if needed.

Reply:
Fine with me.